src/components/WarningAnimateMessage/index.js

```diff
@@ -121,7 +121,7 @@ const styles = StyleSheet.create({
     borderLeftWidth: 10,
     borderTopColor: COLORS.transparent,
     borderRightColor: COLORS.transparent,
-    borderBottomColor: COLORS.concrete,
+    borderBottomColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in `borderBottomColor` from `COLORS.concrete` to `COLORS.gray100` is noted. Again, make sure this color change aligns with the overall design language of your application.

```diff
-    borderBottomColor: COLORS.concrete,
+    borderBottomColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->